src/components/shared/renderers/checkbox-styles.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../../styles/variables.scss';
+
+.checkbox {
+  list-style: none;
+  
+  &__wrapper {
+    align-items: center;
```

Comment:
I think you might need to change this to `flex-start` or maybe even just drop it...
![Screenshot from 2019-10-24 21-05-26](https://user-images.githubusercontent.com/18272064/67535764-0a552d80-f6a2-11e9-822e-7c908e9d2a86.png)

Reply:
😂